tests/scripts/run_models.sh

```diff
@@ -18,10 +18,14 @@ if [[ $ARCH_NAME == "wormhole" ]]; then
 fi
 
 if [[ $ARCH_NAME == "wormhole_b0" ]]; then
-  env pytest models/experimental/mamba/tests/test_full_model.py
-  env pytest models/experimental/mamba/tests_opt/test_full_model.py
-  env pytest models/experimental/mamba/tests/test_benchmarks.py
-  env pytest models/experimental/mamba/tests/test_demo.py
+  env pytest models/demos/mamba/tests/test_mamba_ssm.py
+  env pytest models/demos/mamba/tests/test_mamba_block.py
+  env pytest models/demos/mamba/tests/test_residual_block.py
+  env pytest models/demos/mamba/tests/test_full_model.py::test_inference_loop
```

Comment:
Can you please target only a filename? If there are multiple tests in this file, please move them to a separate file

Reply:
Refractored and added a separate test file